src/components/page/elements/index.js

```diff
@@ -19,6 +19,8 @@ import TableElement from './TableElement'
 import FormElement from './FormElement'
 import MultipleChoiceQuestionElement from './MultipleChoiceQuestionElement'
 import QuizElement from './QuizElement'
+import StrikethroughElement from './StrikethroughElement'
+import HorizontalLineElement from './HorizontalLineElement'
 
 export default {
```

Comment:
quick thought: can't the key (like `h1`, `exercise`, `img`, ...) be a constant in all `Element` classes?
Then we can create an `Element` interface which all element should implement, with a function `getTag()`

Should not be fixed in this pr but it could make it more standardized

Reply:
yes, that would make it more standardized.

for now most of the Elements are just functional components, which would need to be changed to class components. but besides that, no problem

Reply:
added issue #56